### PR TITLE
PT-1765: Validation Assets

### PR DESCRIPTION
**Pull request by ErickLipin**
https://virtocommerce.atlassian.net/browse/PT-1764

### CODE REVIEW

File: src/VirtoCommerce.Platform.Web/Controllers/Api/AssetsController.cs [hunk 1]
```diff
@@ -186,6 +186,9 @@ public async Task<ActionResult<BlobInfo[]>> UploadAssetAsync([FromQuery] string
         [ProducesResponseType(typeof(void), StatusCodes.Status204NoContent)]
         public async Task<ActionResult> DeleteBlobsAsync([FromQuery] string[] urls)
         {
+            if (urls.IsNullOrEmpty())
+                return BadRequest("Please, specify all deleting assets urls.");
```
Comment (akak1977) [inline src/VirtoCommerce.Platform.Web/Controllers/Api/AssetsController.cs:190]: I think it's better to say:
"Please, specify at least one asset URL to delete."

File: src/VirtoCommerce.Platform.Assets.AzureBlobStorage/AzureBlobProvider.cs [hunk 2]
```diff
@@ -121,6 +121,9 @@ public virtual async Task RemoveAsync(string[] urls)
         {
             foreach (var url in urls)
             {
+                if (string.IsNullOrWhiteSpace(url))
+                    continue;
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Assets.AzureBlobStorage/AzureBlobProvider.cs:125]:
```suggestion
            foreach (var url in urls.Where(x => !string.IsNullOrWhiteSpace(x)))
            {
```

**Review by Vectorfield4 — CHANGES_REQUESTED** [review]
Sign CLA

File: src/VirtoCommerce.Platform.Assets.AzureBlobStorage/AzureBlobProvider.cs [hunk 3]
```diff
@@ -119,8 +119,11 @@ public virtual async Task<Stream> OpenWriteAsync(string blobUrl)
 
         public virtual async Task RemoveAsync(string[] urls)
         {
-            foreach (var url in urls)
+            foreach (var url in urls.Where(x => !string.IsNullOrWhiteSpace(x)))
             {
+                if (string.IsNullOrWhiteSpace(url))
+                    continue;
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Assets.AzureBlobStorage/AzureBlobProvider.cs:125]: Redundant check. Remove it.

**Review by Vectorfield4 — CHANGES_REQUESTED** [review]

**Pull request merged**